Dockerfile

```diff
@@ -14,11 +14,12 @@
 
       # Pentaho Directory
       RUN mkdir ${PENTAHO_INSTALL}
-      RUN chmod 775 ${PENTAHO_INSTALL}
+
+      WORKDIR ${PENTAHO_INSTALL}
 
       # Download PDI and extract it
       RUN curl ${PENTAHO_URL} -o $HOME/pdi-ce-${PDI_VERSION}.zip
-      RUN cd $PENTAHO_INSTALL && jar -xvf $HOME/pdi-ce-${PDI_VERSION}.zip
+      RUN jar -xvf $HOME/pdi-ce-${PDI_VERSION}.zip
```

Comment:
warum werden nicht die Dateien in ein Build-Container heruntergeladen und extrahiert und dann in das Image kopiert?